BukkyOmo/Questioner
# #163846763 implement refactoring on code base #163846763

#### What does this PR do?
This PR:
* Implement refactoring.

#### Description of Task to be completed
* refactor tests file.

#### How should this be manually tested?
* Clone this repository
* Check out to <ch-implement-refactoring-on-codebase-#163846763> branch
* Enter user details on postman and test signup and signin.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories? (if applicable)
##163846763
